application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +146,35 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
-}
+
+    private void reminderFailureAction(int id, long channelId, long authorId,
```

Comment:
the name is kinda bad. what about `attemptRetryReminder`

Reply:
just saw this and updated to `attemptRetryReminder`